lib/model.js

```diff
@@ -236,23 +236,21 @@ function conformOptions(options, self) {
     self.$expandAttributes(options);
   }
 
-  if (!options.include) {
-    return;
-  }
-  // if include is not an array, wrap in an array
-  if (!Array.isArray(options.include)) {
-    options.include = [options.include];
-  } else if (!options.include.length) {
-    delete options.include;
-    return;
-  }
+  if (options.include) {
+    // if include is not an array, wrap in an array
+    if (!Array.isArray(options.include)) {
+      options.include = [options.include];
+    }
 
-  // convert all included elements to { model: Model } form
-  options.include = options.include.map(function(include) {
-    include = conformInclude(include, self);
+    // convert all included elements to { model: Model } form
+    options.include = options.include.map(function(include) {
+      return conformInclude(include, self);
+    });
 
-    return include;
-  });
+    if (!options.include.length) {
+      delete options.include;
+    }
+  }
```

Comment:
Did anything actually change here other than changing it away from short circuiting?